src/data-workspace/validation/validation-results-sidebar.test.js

```diff
@@ -196,13 +196,15 @@ describe('ValidationResultsSidebar', () => {
         const overrideOptions = {
             validationRules: () => {
                 return new Promise((resolve, reject) => {
-                    if (count++ === 1) {
-                        return reject(
-                            'server-side error the first time then it will pass on re-run'
-                        )
-                    } else {
-                        return resolve(validationMetadataMockResponse)
-                    }
+                    setTimeout(() => {
```

Comment:
This seemed to be necessary, or else the `running validation`-text would not be found, because it updated instantly.